code/modules/projectiles/guns/projectile/automatic/z8.dm

```diff
@@ -31,6 +31,8 @@
 		list(mode_name="fire grenades", mode_desc="Unlocks the underbarrel grenade launcher", burst=null, fire_delay=null, move_delay=null,  icon="grenade", use_launcher=1)
 		)
 
+	gun_tags = list(GUN_SILENCABLE)
```

Comment:
I'm pretty sure it will mess with the grenade launcher

Reply:
Silenced grenade launcher